pages/common/lpass.md

```diff
@@ -0,0 +1,23 @@
+# lpass
+
+> Command line interface for LastPass Password Manager.
+
+- Login to your LastPass account, by entering your Master Password when prompted:
+
+`lpass login {{username}}`
+
+- Show login status:
+
+`lpass status`
+
+- List all sites grouped by category:
+
+`lpass ls`
+
+- Generate a new password and add to LastPass:
+
+`lpass generate --username {{username}} --url {{gmail.com}} {{gmail.com}} {{password_length}}`
+
+- Show password for a specified entry (usually the website url):
```

Comment:
Since we are making the entry different from the website, let us remove the "usually the website url" part. The user can figure out that he/she can always make the identifier same as the website.

Reply:
I put that in there as I assume most of the passwords in a users vault are created by a browser plugin which uses the domain name as identifier. But I can change if you would like?

Reply:
Hmm ok. But won't it be confusing that the example will show an identifier that is _not_ the website url, but the description mentions that usually it's the website url. That is my only concern.

Reply:
I'm not sure if I understand you fully. 

To clarify, the `lpass generate`-example will create an entry similar to the one a browser-plugin would, with the identifier in this case being `gmail.com`. In the next example where we show the password, we again use the same identifier `gmail.com` to extract the password that was generated.

Reply:
Never mind, I didn't see your other comments before. Have amended the two last ones according to your input.